login.css

```diff
@@ -172,25 +189,23 @@ p{
   	top: 240px;
 }
 
-a{
-	color:black;
-	font-size:18px;
-	text-decoration:none;
-}
 
 #update{
 	color:black;
 	font-size:14px;
 	text-decoration:none;
+	font-family: 'Roboto', sans-serif;
```

Comment:
13/08/2021: I set the fonts for the text for the forgot password link to roboto and because it is normal text I set the font weight to 300.